src/auth/auth-config.ts

```diff
@@ -624,13 +691,28 @@ export class MultiFactorAuthConfig implements MultiFactorConfig {
         this.factorIds.push(AUTH_FACTOR_SERVER_TO_CLIENT_TYPE[enabledProvider]);
       }
     })
+    this.providerConfigs = [];
```

Comment:
Asking for my own clarification - this validation is for the response from the backend and the above code in "validate()" is for user/customer input, correct?

Reply:
Yes